node/engine/chainservice/simulated_backend_chainservice_test.go

```diff
@@ -65,21 +60,52 @@ func TestSimulatedBackendChainService(t *testing.T) {
 		},
 		ChannelNonce:      37140676580,
 		AppDefinition:     bindings.ConsensusApp.Address,
-		ChallengeDuration: 0,
+		ChallengeDuration: 1000,
 		AppData:           []byte{},
 		Outcome:           concludeOutcome,
 		TurnNum:           uint64(2),
 		IsFinal:           true,
 	}
 
-	// Prepare test data to trigger EthChainService
+	challengerSig, err := NitroAdjudicator.SignChallengeMessage(concludeState, Alice.PrivateKey)
```

Comment:
This test now:
* challenges the channel
* then deposits
* then concludes
It's not a typical user flow but it does cover what we want it to.